packaging/test_rule_engine_plugin_metadata_guard.py

```diff
@@ -307,7 +307,8 @@ def test_plugin_cannot_be_bypassed_via_imeta_mod__issue_58(self):
 
                 # Same as above, but with units
                 self.user.assert_icommand(['imeta', 'add', '-C', self.user.session_collection, 'unprotected::withunits', 'v', 'u'])
-                self.user.assert_icommand(['imeta', 'mod', '-C', self.user.session_collection, 'unprotected::withunits', 'v', 'n:irods::withunits', 'u'], 'STDERR', ['CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
+                self.user.assert_icommand(['imeta', 'mod', '-C', self.user.session_collection, 'unprotected::withunits', 'v', 'u', 'n:irods::withunits'], 'STDERR', ['CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
+                self.user.assert_icommand(['imeta', 'mod', '-C', self.user.session_collection, 'unprotected::withunits', 'v', 'n:irods::withunits', 'u:newunits'], 'STDERR', ['CAT_INSUFFICIENT_PRIVILEGE_LEVEL'])
```

Comment:
`u:newunits` -> `u`

We want to capture the fact that the argument positions don't matter to `imeta`.

Reply:
Oh, I see, so it does match the input "units" when placed there. Gotcha, will change!

Reply:
@FifthPotato Just read your comment here: https://github.com/irods/irods_rule_engine_plugin_metadata_guard/pull/69#discussion_r1495052305

Is my comment about changing `u:newunits` to `u` still relevant?

Reply:
Yep! Bad interpretation on my part, so we want to check the input units can be passed out-of-order, not the output units. Should be good now!